### PR TITLE
Fix debian9 failure by updating baseline

### PR DESCRIPTION
**Pull request by danmosemsft**
Fix https://github.com/dotnet/corefx/issues/20217

### CODE REVIEW

**Comment by danmosemsft** [comment]
@dotnet-bot help please

**Comment by danmosemsft** [comment]
@mmitche, @mattgal [added Debian 9 to helix runs ](https://github.com/dotnet/corefx/pull/20196/files) but it's not available here. does this netci.groovy edit accomplish that? Are we getting to a place where distros don't need to be added in both places separately?

**Comment by danmosemsft** [comment]
@dotnet-bot test Innerloop Ubuntu14.04 Debug x64 Build and Test please (https://github.com/dotnet/corefx/issues/20246)

**Comment by tarekgh** [comment]
LGTM

File: netci.groovy [hunk 1]
```diff
@@ -49,7 +51,7 @@ def buildArchConfiguration = ['Debug': 'x86',
                               'Release': 'x64']
 
 def targetGroupOsMapOuterloop = ['netcoreapp': ['Windows 10', 'Windows 7', 'Windows_NT', 'Ubuntu14.04', 'Ubuntu16.04', 'Ubuntu16.10', 'CentOS7.1', 'OpenSUSE13.2', 'OpenSUSE42.1', 
-                                        'RHEL7.2', 'Fedora24', 'Debian8.4', 'OSX10.12', 'PortableLinux'],
+                                        'RHEL7.2', 'Fedora24', 'Debian8.4', 'Debian9.0', 'OSX10.12', 'PortableLinux'],
```
Comment (danmosemsft) [inline netci.groovy]: @mmitche does this make it show up as an optional command in CI jobs?

**Comment by mmitche** [comment]
@chulhul is there a Debian 9 open queue?
@danmosemsft No, since Debian 9 is portable, it's added here. 
 https://github.com/dotnet/corefx/blob/master/buildpipeline/portable-linux.groovy#L58

**Comment by danmosemsft** [comment]
I'll just fix the test for now, and maybe add Debian 9 to jenkins later if I have time later.

**Comment by ChulHul** [comment]
@mmitche I don't think we have an open queue. I'll set one up today or early tomorrow.

**Comment by ChulHul** [comment]
@mmitche queue deployed.

**Pull request merged**